usr/src/man/man3nvpair/nvlist_add_boolean.3nvpair

```diff
@@ -251,6 +251,10 @@ thread can actively change a given \fBnvlist_t\fR at a time. The caller is
 responsible for the synchronization.
 .sp
 .LP
+The name used for any name-value pair must be less than 2^15 bytes in
+length, otherwise EINVAL will be returned.
```

Comment:
@rmustacc Please let me know if you think this is sufficient and accurate enough, or if I should add some more information.

I verified the format manually using `mandoc`:
```
       Multiple threads can simultaneously read the same nvlist_t, but only
       one thread can actively change a given nvlist_t at a time. The caller
       is responsible for the synchronization.


       The name used for any name-value pair must be less than 2^15 bytes in
       length, otherwise EINVAL will be returned.


       The list that is added to the parent nvlist_t by calling
       nvlist_add_nvlist() is copied and thus is not freed when nvlist_free()
       is called on the parent list. To prevent memory leaks, your code needs
       to look like the following (error handling elided for clarity):
```